src/schema.py

```
@@ -833,81 +835,94 @@ class Arguments:
         if existing_user:
             raise GraphQLError("NetID already exists.")
 
+        final_photo_url = None
         if encoded_image:
-            upload_url = os.getenv("DIGITAL_OCEAN_URL")
-            if not upload_url:
-                raise GraphQLError("Upload URL not configured.")
             
-            headers = {"Content-Type": "application/json"}
+            s3 = boto3.client('s3',
+                endpoint_url=os.getenv("DIGITAL_OCEAN_URL"),
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)